src/Components/Artwork/__tests__/Details.test.tsx

```diff
@@ -40,6 +40,14 @@ describe("Details", () => {
       expect(html).toContain("ends in 14d")
     })
 
+    it("shows sale 'timely' data when auction is in a preview state", async () => {
+      const data = cloneDeep(artworkInAuction)
+      data.sale.is_open = false
+      const wrapper = await getWrapper(data)
+      const html = wrapper.html()
+      expect(html).toContain("ends in 14d")
```

Comment:
I thought for auctions in preview we would show `starts in 1d` instead of the end time. Any case this state got missed in the timer?

Reply:
I don't think so. The logic for this label lives entirely in Metaphysics as `display_timely_at`, the comp for this labeling had `ends in ...` for that case. The only `starts..` case was for live auctions.